mlrun/projects/project.py

```diff
@@ -483,6 +498,7 @@ def __init__(
         self._function_definitions = {}
         self.functions = functions or []
         self.disable_auto_mount = disable_auto_mount
+        self.user_project = user_project
```

Comment:
When we talked about it offline we said we **will not** make it a property of the project, but only an argument in the project object instantiation methods (`get_or_create_project`, `load_project``new_project`)